pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  # This mimics ../../../development/interpreters/python/wrap.sh
+  postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
```

Comment:
@FRidh `wrapPythonPrograms` does but `wrapPythonProgramsIn` doesn't. We need the latter because `/libexec` also needs to be wrapped (perhaps we could just add it to `wrapPythonPrograms` instead?).